mlflow/utils/databricks_utils.py

```diff
@@ -462,11 +465,20 @@ def get_workspace_info_from_databricks_secrets(tracking_uri):
     return None, None
 
 
-def _fail_malformed_databricks_auth(profile):
+def _fail_malformed_databricks_auth(tracking_uri):
     raise MlflowException(
-        "Got malformed Databricks CLI profile '%s'. Please make sure the "
-        "Databricks CLI is properly configured as described at "
-        "https://github.com/databricks/databricks-cli." % profile
+        f"Reading databricks credential configuration failed using tracking URI {tracking_uri}, "
+        "Please ensure that you installed 'databricks-sdk' library, set correct tracking "
+        "URI and set up databricks authentication configuration correctly. "
+        "The available tracking URI can be either 'databricks' "
+        "(using 'DEFAULT' authentication profile) or 'databricks://{profile}'. "
+        "To set up databricks authentication configuration, you can set environmental "
+        "variables DATABRICKS_HOST + DATABRICKS_TOKEN, or set environmental variables "
+        "DATABRICKS_HOST + DATABRICKS_CLIENT_ID + DATABRICKS_CLIENT_SECRET, or you can "
+        "edit '~/.databrickscfg' file to set host + token or host + client_id + client_secret "
+        "for specific profile section. For details of these authentication types, please "
+        "refer to document 'https://docs.databricks.com/en/dev-tools/sdk-python.html"
+        "#use-the-databricks-sdk-for-python-from-a-databricks-notebook'."
```

Comment:
I think this is too long. Can we just reference https://docs.databricks.com/en/dev-tools/sdk-python.html#use-the-databricks-sdk-for-python-from-a-databricks-notebook?

Reply:
can we link https://docs.databricks.com/en/dev-tools/auth/index.html#unified-auth?

Reply:
this page looks good! https://docs.databricks.com/en/dev-tools/auth/index.html#unified-auth